src/servers/src/http/influxdb.rs

```diff
@@ -39,13 +44,29 @@ pub async fn influxdb_health() -> Result<impl IntoResponse> {
     Ok(StatusCode::OK)
 }
 
+fn is_gzip(headers: &HeaderMap) -> Result<bool> {
+    match headers
+        .get("Content-Encoding")
+        .map(|val| val.to_str().context(InvisibleASCIISnafu))
+        .transpose()?
+    {
+        None | Some("identity") => Ok(false),
```

Comment:
Should the case sensitivity of "Content-Encoding" be considered? For example, the value can be "Identity" or "GZIP", etc.?

Reply:
https://stackoverflow.com/questions/10953635/are-the-http-connection-header-values-case-sensitive

HeaderValue SHOULD BE case-insensitive. But the implementation does. I'll use `to_lowercase` here.